src/main/java/com/sap/sgs/phosphor/fosstars/data/github/NvdEntryMatcher.java

```diff
@@ -173,4 +184,40 @@ private static boolean match(String one, String two) {
     two = two.toLowerCase().trim();
     return one.contains(two) || two.contains(one);
```

Comment:
Is this method still used?

Reply:
I am thinking about how to better use this. Why it is compared with `contains` when comparing organization and vendor name? Was there a specific reason?

Reply:
No strong reasons.